addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -34,7 +34,9 @@
         <!-- New report layout for din5008 format -->
         <template id="external_layout_din5008">
             <div>
-                <div t-attf-class="header din_page o_company_#{company.id}_layout #{'din_page_pdf' if report_type == 'pdf' else ''}">
+                <t t-set="is_din_paperformat"
+                    t-value="company.paperformat_id in [env.ref('l10n_din5008.paperformat_euro_din_a'), env.ref('l10n_din5008.paperformat_euro_din')]" />
```

Comment:
use `raise_if_not_found`. Else if someone decided to delete it, they would get a traceback